packages/integrations/src/vue.ts

```diff
@@ -426,7 +426,7 @@ export class Vue implements Integration {
 
       if (this._options.logErrors) {
         if (this._options.Vue.util) {
-          this._options.Vue.util.warn(`Error in ${info}: "${error.toString()}"`, vm);
+          this._options.Vue.util.warn(`Error in ${info}: "${error && error.toString()}"`, vm);
```

Comment:
I don't know Vue, but will there ever be `info` if there is no `error`? What about `vm`?

In any case, it might be nice to provide the user with the information that the error is null rather than just now showing anything. That way, they know it's not the reporting that's broken.
```suggestion
          this._options.Vue.util.warn(`Error in ${info}: "${error && error.toString() || 'null'}"`, vm);
```

Reply:
```
Error in foo: null
```

Looks strange to me, why would there be a warning in the first place.

Reply:
Worse than `Error in foo:`, though? Maybe `Error in foo: <null>` is better?